model/godleyTable.cc

```diff
@@ -211,20 +213,64 @@ string GodleyTable::rowSum(int row) const
           else
             sum[fc.name]+=fc.coef;
         }
+      } else   // display numerical sum of stock vars in first cell of last column. for ticket 1285
```

Comment:
It might be simpler just to have a separate loop summing up the values of the stock variables for row 0, rather than this rather tortuous dealing with row 0 in a separate branch in the loops dealing with the other rows.

Reply:
Done, but values of stock vars did not display well. Numerical inaccuracies grew beyond 1.1e-15 after integrating a model for sufficient time. This caused values such as 1e-14 and greater to be displayed all the time in the first cell of the A-L-E column.